docs/ops/indexing.rst

```diff
@@ -200,3 +200,124 @@ Once you have downloaded some data, it will need :ref:`metadata preparation
 <prepare-scripts>` before use in the Data Cube.
 
 
+Indexing Data on Amazon(AWS S3)
+================
+Cloud storage is a sought after feature for most storage platforms. Options currently exist that allow for a users to store, index, and retrieve data from Amazon S3 buckets using the open data cube. The following sections outline this process.  
+
+Configuring AWS CLI Credentials
+-----------------------------
+
+Install the AWS CLI package and configure it with your Amazon AWS credentials. For a more detailed tutorial on AWS CLI configurations, visit the official AWS docs  The
+only two fields required to be configured are the ``Access Key``, and
+``Secret Access Key``. These keys can be found on your AWS login
+security page. Try not to lose your ``Secret Access Key`` as you will
+not be able to view it again and you will have to request a new one.
+
+::
+
+    pip install boto3 ruamel.yaml 
+    sudo apt-get install awscli -y
+    aws configure
+
+Add the ca-certificates requisite for S3 indexing and export them to the
+environment variable the data cube will look for. If you forget this
+step you will see an error upon attempting to load the indexed dataset.
+
+::
+
+    sudo apt-get install ca-certificates
+    export CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
+
+You may want to add the line
+``export CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt`` to your
+``.bashrc`` file to make these changes permanent.
+
+
+Download Indexing Scripts
+-----------------------------
+
+
+In order to utilize the convenience of S3 indexing, we must retrieve
+scripts necessary for S3 indexing. The direct links are provided below
+since, at the time of this document, they are not all included in the
+latest release (1.6.1).
+
+::
+
+    cd ~/Datacube
+    mkdir -p S3_scripts
+    cd S3_scripts
+    wget https://raw.githubusercontent.com/opendatacube/datacube-core/develop/datacube/index/hl.py
```

Comment:
@otto-AMA 
This seems like a very fragile way of setting up prepare scripts? Why is this neccessary? You need datacube installed to perform indexing, why copying `hl.py` from github is needed?

Reply:
The script linked below imports hl from `datacube.index`, so it won't use local copy anyway, seems like leftover instruction from the past that is no longer relevant.

Reply:
Should we set up a new PR to resolve this now, while we're thinking about it?

Reply:
@alexgleith or @otto-AMA can either of you confirm that copying `hl.py` is not necessary (I believe it is not necessary) and if yes do new PR that removes unneeded instruction line from the docs? If this is being done in some docker/bash script file then that should be fixed too.

Reply:
Hey @otto-AMA do you think you could tackle this now, as it'll be easier while it's fresh!